code/modules/reagents/reagent_containers/patch.dm

```diff
@@ -13,13 +13,42 @@
 	..()
 	icon_state = "bandaid" // thanks inheritance
 
-/obj/item/weapon/reagent_containers/pill/patch/afterattack(obj/target, mob/user , proximity)
-	return // thanks inheritance again
+/obj/item/weapon/reagent_containers/pill/patch/attack(mob/living/carbon/human/M, mob/user)
+	if(istype(M))
+		var/obj/item/organ/limb/affected = canconsume(M, user)
+		if(affected && affected.status == ORGAN_ORGANIC)
+			user.visible_message("<span class='notice'>[user] manages to [apply_method] [src] on [M].</span>", \
```

Comment:
Currently only 3 healing chems have any effect, anything else will do exactly nothing
Yes i know its bad, yes this will most likely be changed